Huion_GC610/index.md

```diff
@@ -0,0 +1,20 @@
+---
+VID: "256c"
+PID: "006e"
+vendor: Huion
+product: HGC610
+image: Huion_HGC610
+report_rate: 233
+working_area:
+    width: 10
+    height: 6.25
+    resolution: 5080
+pen:
+    pressure_levels: 8192
+frame_controls: 2-4 fingers 6 programmable express keys 16 programmable soft keys
```

Comment:
Let's not include the "16 programmable soft keys", as they're not actual physical controls, which tablet reports. Also "2-4 fingers" belongs to the multi-touch pad capabilities. Let's just write "multi-touch pad" instead for now. Lastly, we don't need to write "programmable", that's part of the software, here we're concerned with the hardware, so just "6 buttons". All-in-all, this boils down to: "multi-touch pad, 6 buttons".